src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -199,8 +199,8 @@ protected virtual void PreConfigure(Schema schema)
 
         protected virtual IGraphType GetType(string name)
         {
-            _types.TryGetValue(name, out IGraphType type);
-            return type;
+            _types.TryGetValue(name, out var type);
+            return type!;
```

Comment:
Seems like either it should be nullable return value or else use the Item property so an exception is thrown if the key doesn’t exist. Probably do the former and then change to the latter in a separate PR